src/normalize-model-and-view.js

```diff
@@ -25,6 +27,27 @@ export function getModelPath (reference) {
   return path
 }
 
+function appendModelPath (modelPath, id, internal) {
+  const addedModelPath = getModelPath(id)
+  if (internal) {
+    return `${modelPath}.properties._internal.${addedModelPath}`
+  }
+  if (modelPath === '') {
+    return addedModelPath
+  }
+  return `${modelPath}.${addedModelPath}`
+}
+
+function expandCell (cell, cellDefinitions) {
+  cell = _.clone(cell)
+  while (cell.extends) {
+    const extendedCell = cellDefinitions[cell.extends]
```

Comment:
Is there prior validation that this `extends` is valid? I come across invalid schema all the time due to mispelling and hate it when I have to dig through the source to figure out it was just a mispelling.